chicago/views.py

```diff
@@ -332,6 +332,43 @@ def get_context_data(self, **kwargs):
             .order_by("-last_action")[:10]
         )
 
+        attendance = []
+        events = []
+        # get all events across all memberships tenures
+        for membership in person.current_memberships.all():
+            events.extend(
+                ChicagoEvent.objects.filter(
```

Comment:
this query can be something like this:

from django.db.models import Prefetch

```python
ChicagoEvent.objects
    .filter(
        participants__organization=membership.organization,
        start_date__gte=membership.start_date)
    .filter(start_date__gte=current_legislative_session.start_date)
    .filter(participants__entity_type='person')
    .distinct()
    .prefetch_related('participants')
```
this will give you all events where

* the person was a member
* in the current legislative session
* where attendance was taken.
* prefetches the attendance